XCUITests/TopTabsTest.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+let url = "www.mozilla.org"
+let urlLabel = "Internet for people, not profit — Mozilla"
+let urlValue = "mozilla.org"
+
+let urlYah = "www.yahoo.com"
+let urlLabelYah = "Yahoo"
+let urlValueYah = "yahoo"
+
+let urlExample = "example.com"
+
+class TopTabsTest: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testAddTabFromSettings() {
+        navigator.goto(NewTabMenu)
+        app.collectionViews.cells["NewTabMenuItem"].tap()
```

Comment:
@npark-mozilla is correct here: there are multiple edges going to the same node here. The difference is that Closing the Menu takes you back to the existing new tab screen.

I think what we're edging towards is that the actual states should be `ScreenNodes` that we create in `FxScreenGraph`, and that actions (e.g. `createNewTab` and `openURL` should be methods on `Navigator`.

Each of these actions should almost certainly start with a `goto` call.